console/src/components/Services/Events/Common/Components/utils.tsx

```diff
@@ -30,3 +31,21 @@ export const getEventDeliveryIcon = (delivered: boolean) => {
 export const getInvocationLogStatus = (status: number) => {
   return status < 300 ? <CheckIcon /> : <CrossIcon />;
 };
+
+export const getExpanderButton = (isExpanded: boolean) => (
+  <Button
+    color="white"
+    size="xs"
+    title={isExpanded ? 'Collapse row' : 'Expand row'}
```

Comment:
There's an _expand_ icon for collapsing and _collapse_ icon for expanding. 
<img width="300" alt="Screenshot 2020-09-14 at 22 45 22" src="https://user-images.githubusercontent.com/9019397/93136314-fb250700-f6db-11ea-9970-c1054ebb20bf.png">
<img width="203" alt="Screenshot 2020-09-14 at 22 45 36" src="https://user-images.githubusercontent.com/9019397/93136326-037d4200-f6dc-11ea-8a6d-6ab670ed8026.png">

Reply:
I could not really reproduce this anywhere in `Event Triggers`, `Cron Triggers`, `One-Off Scheduled Events`. 

I see this everywhere:
![image](https://user-images.githubusercontent.com/13602716/93374591-8a652280-f874-11ea-8698-90dbdf590059.png)

Can you please help me with pinpointing the issue, so that I could rectify?  All other comments are addressed.